web/src/test/java/se/inera/intyg/webcert/web/web/controller/integrationtest/facade/LisjpIT.java

```diff
@@ -591,6 +602,65 @@ void shallReturnDraftOfCurrentVersion() {
             );
         }
 
+        @Test
+        @DisplayName("Shall return draft with FMB warning")
+        void shallReturnDraftWithFMBWarning() {
+            final var icd10Codes = new String[]{"F500"};
```

Comment:
Suggest you refactor the test data creation by Extract method. 

As a general rule, it can be good to work to reduce method-size and try to wrap code-blocks that has a specific purpose e.g. create test data in this case. Readability is greatly improved.